### DISCUSSION TITLE
Disable font antialiasing

### DISCUSSION THREAD

**Discussion by j-gynn** [MyreMylar/pygame_gui#616] [Q&A] [answered]
Really appreciate the work going into this library, it's made our lives so much easier. I'm currently working on a project that has a somewhat retro styling, and as a result we're using a font that we would really like to keep crisp. I know this is possible with pygame by setting the antialiasing property when calling Font.render, but is there an equivalent in pygame_gui? If this doesn't exist, is there an easy way to expose that functionality aside from having to subclass half of the library?

I haven't been able to dig up anything about font antialiasing in the documentation or on GitHub but please point me if I've missed something. Thank you!

**Comment by j-gynn** [comment]
Update: I have located the reason our fonts are aliased (I believe). The preload_fonts function doesn't seem to be working for the latest implementation I've been working on, so I'll dig through that to see if there's a cause there.

**Comment by MyreMylar** [comment, marked as answer]
There was some work put into this. There is an 'antialiased' theming
parameter you can set for elements with font theming. Just set it to "0" to
turn off antialiasing on e.g. a button.

There isn't a convenient way to tag some html text in a text box as
antialiased yet though. It could fairly easily be added to the font tag
parser as a new attribute if that is what you are looking for.

**Comment by j-gynn** [comment]
Got it working now. Seems to be undocumented that you have to specify it in the `font` section for each theme that uses it, rather than having the font globally be non-antialiased... unless I have also missed that haha
